modules/tensor_mechanics/src/materials/ComputeThermalExpansionEigenstrainBase.C

```diff
@@ -23,6 +23,9 @@ ComputeThermalExpansionEigenstrainBaseTempl<is_ad>::validParams()
   params.addParam<bool>("use_old_temperature",
                         false,
                         "Flag to optionally use the temperature value from the previous timestep.");
+  params.addParam<MaterialPropertyName>("alpha_bar_name",
```

Comment:
should be alpha_bar or `mean_thermal_expansion_name`?

Reply:
Please do away with this "alpha bar" stuff. "mean thermal expansion (coefficient)" is much clearer